ts/features/wallet/onboarding/bancomat/screens/add-pans/AddBancomatComponent.tsx

```diff
@@ -39,6 +44,13 @@ const styles = StyleSheet.create({
   title: { lineHeight: 33, alignSelf: "flex-start" },
   flexStart: { alignSelf: "flex-start" }
 });
+
+const isBancomatBlocked = (pan: Card) =>
+  fromNullable(pan.validityState).fold(
```

Comment:
I guess you could avoid fromNullable
```suggestion
  props.bancomat.validityState === ValidityStateEnum.BR
```
if it is `undefined` it is different from `ValidityStateEnum.BR`